blog/tidal_profile_ndr_brt.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Tidal Profile - ndr_brt
+---
+
+| Tidal Cyclist  |     |
+| --------:    | :---------- |
+| aka    | ndr_brt |
+| Location | Italy |
+| Years with Tidal | 4  yrs |
+| Other LiveCoding env | Hydra, Supercollider, Threnoscope, ByteBeat |
+| Music available online | [Bandcamp](https://umanoudito.bandcamp.com/) |
+| Code online | [GitHub](https://github.com/ndr-brt/sc-adente) |
+| Other music/audio sw | sox, ffmpeg, Ardour, Audacity |
+
+## Livecoding  
+
+**What do you like about livecoding in Tidal? What inspires you?**   
+When I met it for the first time everything was a wow, the cycle concept, function composition, mini-notation, patternization...
+Nowadays I'm still able to find inspiration watching other people livecoding or reading the posts on tidal club, especially when there are custom functions listed.
+
+
+**How do you approach your livecoding sessions?**  
+I always try to start from scratch, when I code alone I usually focus on a single function and try to get everything out of it, while I'm in front of an audience I just go with the flow.
+
+**What functions and coding approaches do you like to use?**
+I'm a huge fan of `superimpose` (used with the `si` shortand), especially mixed with `hurry`, in my "single sample runs" I create layers of the same sample playing at different speed/density to create rhythm and melodic patterns, I often try to escape from this mindset but at the end I fall into it most of the times.
+
+**Do you use Tidal with other tools / environments?**
+I tried it to control some drum machines circuit bent by me but at the end I find the hardware overcomplicated and I prefer to play soft-synths, especially Supercollider: everything in a box and controllable with the keyboard.
+I used Tidal also to [draw stuff with p5 during some sessions](https://www.youtube.com/watch?v=lbUCSVC4GPs).
+
+## Tidal Contributions  
+
+**How do you contribute to Tidal Cycles? What have you worked on?**
+I learned Haskell only to contribute to Tidal, I'm passionate about reading code and get the insights of the software, on the main codebase I solved some bugs and added some feature mainly in the mini-notation section. I also took care of the migration from Travis CI to GitHub Actions.
+At a certain point I noticed that the Atom Plugin was practically unmaintained so I proposed to be its maintainer, and I brought it back on track. Now Atom has been disbanded but luckily the Pulsar community is vibrant and the Tidal plugin is already fully compatible with it. It was a pretty satisfying migration.
```

Comment:
I suggest adding a little more that highlights more of what you contribute: I manage the [plugin code](https://github.com/tidalcycles/pulsar-tidalcycles) and keep track of publishing any changes to the plugin.